jason/tests/peer/mod.rs

```diff
@@ -2,15 +2,24 @@
 
 mod media;
 
-use std::rc::Rc;
+use std::{cell::RefCell, rc::Rc};
+
+use futures::{
+    future::{self, IntoFuture},
+    sync::mpsc::{unbounded, UnboundedReceiver},
+    Future, Stream,
```

Comment:
If trait doesn't used by name explicitly, you should not pollute the current namespace and import it via `as _`.